zigpy/ota/providers.py

```diff
@@ -593,6 +593,7 @@ async def _load_index(
             )
 
 
+
```

Comment:
Can you remove this added space again?

Reply:
OK

Reply:
Ok, now you're removing an empty line instead of adding an additional one. Please confirm the diff is the same for these empty lines.